m3/benchmark-tests.txt

```diff
@@ -0,0 +1,52 @@
+Benchmark                                                                                  Mode  Cnt      Score     Error   Units
+M3ReporterBenchmark.reportCounterBenchmark                                                thrpt   10    392.408 ± 172.561  ops/ms
```

Comment:
Let's also capture other benchmark values (other than GC related)

Reply:
JMH only allows to run one profiler at a time. We can change the gradle task and pass different profilers as a param. But I'd prefer doing it incrementally in another PR